fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractQueryImmunizationTest.java

```diff
@@ -6,13 +6,6 @@
 
 package com.ibm.fhir.persistence.test.common;
 
-import static org.testng.AssertJUnit.assertEquals;
-import static org.testng.AssertJUnit.assertNotNull;
-
```

Comment:
We should revisit this class, not reoganize the imports

Reply:
make sense, kept it untouched for now.